src/api/routes.py

```diff
@@ -15,9 +15,15 @@ def get_masq(
     ua: Union[bool, None] = True,
     rf: Union[bool, None] = False,
     hd: Union[bool, None] = False,
+    count: Union[int, None] = 1,
 ):
     logger.info(f"Request: [{ua=} {rf=} {hd=}]")
-    response = masq(ua, rf, hd)
+    if(count > 1):
+        response = [masq(ua, rf, hd)]
+        for _ in range(count - 1):
+            response.append(masq(ua, rf, hd))
+    else:
+        response = masq(ua, rf, hd)
```

Comment:
I like the approach here for backwards compatibility, it's a good workaround - however see my longer comment on plans to change the API paths, which would mean we can make this easier on ourselves and replace lines `21` to `26` with a list comprehension:

```response = [masq(ua, rf, hd) for _ in range(count)]```